Sources/UIImageExtensions.swift

```diff
@@ -9,6 +9,12 @@
 import UIKit
 
 extension UIImage {
+    
+    /// EZSE: Returns base64 string
+    var base64: String {
+        return UIImageJPEGRepresentation(self, 1.0)!.base64EncodedString()
```

Comment:
I would avoid network IO if I could. Lets keep a downloaded image in some testdata folder or something of that order. Or just on charizard.png.